docs/user-guide/install-zowe-pswi-deployment.md

```diff
@@ -0,0 +1,56 @@
+# Install Product Software Using z/OSMF Deployments
+
+As a systems programmer, your responsibilities include installing product software in your z/OS environment. After the portable software instance or software instance is registered in z/OSMF, you can use z/OSMF Deployments to install the product software and create the product data sets (global, CSI, target libraries, and distribution libraries) for the new software instance. The deployment jobs create a copy of the source product data sets to create the product target runtime environment. Creating a copy of the SMP/E target data sets keeps the SMP/E environment clean and it also isolates the product runtime environment for maintenance activities. You can also perform z/OSMF workflows to customize the SMP/E data sets, mount UNIX System Services (USS) files if necessary, and configure the new software instance on the target system.
+
+To install Zowe PSWI using z/OSMF and make the product software available for use on a system by users and other programs, define a new deployment. This step defines the SMP/E environment name and the prefix of the CSI data set in z/OSMF. You also specify data set allocation parameters for all SMP/E data sets, target libraries, and distribution libraries. To define a new deployment, complete the deployment checklist (specify the USS path, DSN, VOLSERs), and submit the deployment jobs through the z/OSMF user interface. When the deployment is complete, you have a source and target copy of the software.
+
+For more information about these tasks, see [Deploying software](https://www.ibm.com/docs/en/zos/2.4.0?topic=task-deploying-software) in the IBM documentation.
+
+Subsequent maintenance activities for the product update the SMP/E environment without affecting your active product runtime environments. You decide when to redeploy the maintenance-updated SMP/E target data sets to each of the product runtime environments.
+
+Before you begin this process, review the installation best practices and verify that you meet the installation requirements in Prepare for Installation.
```

Comment:
I think this Prepare for Installation either needs a link or more specification as to which article you are referencing.

Reply:
removed - these refer to absent pages in the template